test/fuzzer/pedro/create_replace_table_union.test

```diff
@@ -15,11 +15,17 @@ loop i 0 10
 
 statement error
 CREATE OR REPLACE TABLE t1 AS (SELECT * FROM range(10) ORDER BY 1) UNION ALL (SELECT * FROM range(10) ORDER BY 1);
+----
+Catalog Error: t1 is not an table
```

Comment:
Was just. running these tests and they seem to pass. I might be missing something but I don't see why these tests should fail

Reply:
I wonder what Postgres behavior is, but these fail because we're trying to create or replace a table, but `t1` is a view.

Reply:
> The CREATE OR REPLACE statement doesn’t work for tables in PostgreSQL, unlike other database systems

Ah.. interesting

Reply:
SQLite also doesn't seem to support this syntax

Reply:
Ah, yea. Must have missed that. There is also a loop, but each iteration doesn't seem to test a different scenario. Don't think that is important though.